packages/core/src/Stores/ui-store.js

```diff
@@ -227,6 +227,30 @@ export default class UIStore extends BaseStore {
         return !!this.account_switcher_disabled_message;
     }
 
+    @action.bound
+    filterNotificationMessages() {
+        const new_app_routing_history = this.root_store.common.app_routing_history.slice();
+        const current_platform = getPlatformHeader(new_app_routing_history);
+
+        this.notifications.forEach(notification => {
+            if (notification.platform && !notification.platform.includes(current_platform)) {
+                if (notification.is_disposable) {
+                    this.notifications = this.notifications.filter(
+                        check_notification =>
+                            check_notification.platform.includes(current_platform) ||
+                            check_notification.platform === undefined
+                    );
+                } else {
+                    this.notifications.filter(
```

Comment:
Nothing is happening with this newly created array. Should it be assigned to `this.notifications`?

Reply:
You’re right. This has been changed.